purchase_request/security/purchase_request.xml

```diff
@@ -0,0 +1,91 @@
+<openerp>
+<data noupdate="0">
+
+    <record model="ir.module.category" id="module_category_purchase_request">
+        <field name="name">Purchase Request</field>
+        <field name="parent_id" ref="base.module_category_purchase_management"/>
+        <field name="sequence">10</field>
+    </record>
+
+    <record id="group_purchase_request_user" model="res.groups">
+        <field name="name">Purchase Request User</field>
+        <field name="users" eval="[(4, ref('base.group_user'))]"/>
+        <field name="category_id" ref="module_category_purchase_request"/>
+    </record>
+
+    <record id="group_purchase_request_manager" model="res.groups">
+        <field name="name">Purchase Request Manager</field>
+        <field name="implied_ids" eval="[(4, ref('purchase_request.group_purchase_request_user'))]"/>
+        <field name="category_id" ref="module_category_purchase_request"/>
+    </record>
+
+</data>
+<data noupdate="0">
+
+    <record model="ir.rule" id="purchase_request_comp_rule">
+        <field name="name">Purchase Request multi-company</field>
+        <field name="model_id" ref="model_purchase_request"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),
+            ('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+
+    <record model="ir.rule" id="purchase_request_line_comp_rule">
+        <field name="name">Purchase Request Line multi-company</field>
+        <field name="model_id" ref="model_purchase_request_line"/>
+        <field name="global" eval="True"/>
+        <field name="domain_force">['|',('company_id','=',False),
+            ('company_id','child_of',[user.company_id.id])]</field>
+    </record>
+
+    <record id="purchase_request_followers_rule" model="ir.rule">
+        <field name="name">Follow Purchase Request</field>
+        <field name="model_id" ref="model_purchase_request"/>
+        <field name="groups" eval="[(6,0, [ref('group_purchase_request_user')])]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="False"/>
+        <field name="perm_create" eval="False"/>
+        <field name="perm_unlink" eval="False"/>
+        <field name="domain_force">['|',('requested_by','=',user.id),
+                                        ('message_partner_ids', 'in', [user.partner_id.id])]</field>
+    </record>
+
+    <record id="purchase_request_modify_rule" model="ir.rule">
+        <field name="name">Modify Purchase Request</field>
+        <field name="model_id" ref="model_purchase_request"/>
+        <field name="groups" eval="[(6,0, [ref('group_purchase_request_user')])]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+        <field name="domain_force">[('requested_by','=',user.id)]</field>
+    </record>
```

Comment:
Should add a rule that disables this one for Purchase Request Manager.
Currently, managers don't see all the requests